readthedocs/projects/static-src/projects/js/import.js

```diff
@@ -282,6 +282,8 @@ function ProjectImportView(instance, config) {
             .then(function (data) {
                 self.get_organizations();
                 self.get_accounts();
+                // re-sync
```

Comment:
It's not a re-sync but a filter by previously selected filter

Reply:
But this is a _hack_ to trigger the re-sync

Reply:
yeah, you are right. So, maybe it's better to improve the comment explaining why we are doing this here and how it works behind the scenes.